src/containers/ProfilePage/index.jsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { Container, Row } from "react-bootstrap";
+import UserDetail from "../../components/UserDetail";
+import UserEmail from "../../components/UserEmail";
+import UserPassword from "../../components/UserPassword";
+import defaultProfileImage from "../../images/defaultProfileImage.png";
+import "./index.scss";
+
+function getArrowClass(selectedElm, currentElm) {
+  return selectedElm === currentElm ? "arrow down" : "arrow right";
+}
+
+function checkCollapseClass(selectedElm, currentElm) {
+  if (selectedElm === currentElm) return "";
+
+  return `collapsed`;
+}
+
+export default function ProfilePage() {
+  const [selectedElement, setSelectedElement] = React.useState("userDetail");
+
+  return (
+    <Container fluid="md" className="profileContainer">
+      <Row className="profilePortrait">
+        <h2>Profile</h2>
+        <div className="userPortrait">
+          <img src={defaultProfileImage} alt="User portrait" />
+        </div>
+      </Row>
+      <Row className="section userDetail">
```

Comment:
I don't see class section being used somewhere

Reply:
done! Thank you